contributions.md

```diff
@@ -0,0 +1,41 @@
+# Contribution Guide
+
+- [Bug Reports](#bug-reports)
+- [Core Development Discussion](#core-development-discussion)
+- [Which Branch?](#which-branch)
+- [Pull Request Guideline](#pull-request-guideline)
+
+<a name="bug-reports"></a>
+## Bug Reports
+
+To encourage active collaboration, we strongly encourages pull requests, not just bug reports. "Bug reports" may also be sent in the form of a pull request containing a failing test.
+
+However, if you file a bug report, your issue should contain a title and a clear description of the issue. You should also include as much relevant information as possible and a code sample that demonstrates the issue. The goal of a bug report is to make it easy for yourself - and others - to replicate the bug and develop a fix.
+
+Remember, bug reports are created in the hope that others with the same problem will be able to collaborate with you on solving it. Do not expect that the bug report will automatically see any activity or that others will jump to fix it. Creating a bug report serves to help yourself and others start on the path of fixing the problem.
+
+The developersnepal source code is managed on Github.
+
+<a name="core-development-discussion"></a>
+## Core Development Discussion
+
+You may propose new features or improvements of existing application behavior in the Developers Nepal Internals [slack #request channel](https://developersnepal.slack.com). If you propose a new feature, please be willing to implement at least some of the code that would be needed to complete the feature.
```

Comment:
By feature I mean something that doesn't exist on the site yet. It could be either an enhancement to the existing parts or a completely new section/page.